datalad/core/local/tests/test_run.py

```diff
@@ -252,7 +252,7 @@ def test_run_from_subds_gh3551(path):
     assert_repo_status(ds.path)
     subds = Dataset(op.join(ds.path, subds_path))
     ok_exists(op.join(subds.path, "f"))
-    if not on_windows:  # FIXME
+    if ds.repo.is_managed_branch():  # FIXME
```

Comment:
Assuming this is indeed an adjusted branch thing, not a windows thing, shouldn't the `not` stay?